study-builder/studies/osteo/parental-consent.conf

```diff
@@ -209,8 +209,8 @@
           "blockType": "GROUP",
```

Comment:
This kind of changes sometimes make it hard to review PR. try not to prettify already existing code if it's not crucial.

Reply:
Okay